data/hails.txt

```diff
@@ -329,13 +337,19 @@ phrase "friendly civilian"
 		"giving their money to"
 		"paying to listen to"
 		"so in love with"
+		"streaming"
+		"on about"
+		"obsessed with"
+		"blinded by"
```

Comment:
Doesn't work: "...my crew is blinded by that new band..."

Reply:
It's not literally "blind" all they can see is the jonas brothers.

Reply:
I see what you're going for, but it doesn't really work in this context.